sysutils/erofs-utils/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cgit 1.0
+
+cgit.setup          git.kernel.org/pub/scm/linux/kernel/git/xiang/ erofs-utils 1.8.9 v
+revision            1
```

Comment:
`revision` should be zero for a new port.

`revision` gets incremented only when you are making changes to an existing port that doesn't change its current version number.

Reply:
@herbygillot fixed! thanks for your reminder